.golangci.yml

```diff
@@ -58,9 +50,11 @@ issues:
     - text: "ST1016:"
       linters:
         - stylecheck
-    - text: "SA1019: codec.LegacyAmino is deprecated"
+    - path: "migrations"
+      text: "SA1019:"
```

Comment:
@okwme not quiet sure  i understand this bit. is the `text` field used as a regex? if so we are changing it?

Reply:
The sdk's config is why this isn't firing the linter:

https://github.com/cosmos/cosmos-sdk/pull/11453

Reply:
I don't think this is needed. @faddat is right in why it's not firing

Reply:
@marbar3778 can you make changes or specific suggestions of what is needed to change before merge?